install/kubernetes/cilium/values.yaml

```diff
@@ -417,6 +417,9 @@ encryption:
   # -- Enable transparent network encryption.
   enabled: false
 
+  # -- Encryption method. Can be either ipsec or wireguard.
+  type: ipsec
+
   # -- Name of the key file inside the Kubernetes secret configured via secretName.
   keyFile: keys
```

Comment:
minor nit: Can we add a comment or something to this and the following ipsec-specific values that it only applies to ipsec?

Reply:
Right, sure!

Reply:
Looks good, thanks! :pray:

Reply:
(I also updated the `cilium-agent-daemonset.yaml` template as just found I missed it)